application/src/main/java/org/thingsboard/server/service/entitiy/queue/DefaultTbQueueService.java

```diff
@@ -56,6 +57,7 @@ public Queue saveQueue(Queue queue) {
             oldQueue = queueService.findQueueById(queue.getTenantId(), queue.getId());
         }
 
+        checkQueueName(queue.getName());
```

Comment:
Please move the check to org.thingsboard.server.dao.service.validator.QueueValidator